ui/admin/tests/acceptance/users/list-test.js

```diff
@@ -27,11 +26,12 @@ module('Acceptance | users | list', function (hooks) {
       org: null,
     },
     users: null,
+    orgScope: null,
   };
 
   hooks.beforeEach(function () {
     instances.scopes.global = this.server.create('scope', { id: 'global' });
-    orgScope = this.server.create(
+    urls.orgScope = this.server.create(
```

Comment:
`urls` is meant to contain the URLs (strings) of the routes used in the test.  Model instances such as the one created here may be contained by `instances`.

Reply:
updated to instances!